x-pack/plugins/apm/public/components/app/settings/general_settings/index.tsx

```diff
@@ -63,6 +65,27 @@ export function GeneralSettings() {
 
   return (
     <>
+      <EuiCallOut
+        title={
+          <FormattedMessage
+            id="xpack.apm.apmSettings.kibanaLink"
+            defaultMessage="{link} to find more options on Kibana advanced settings."
+            values={{
+              link: (
+                <EuiLink
+                  href={application.getUrlForApp('management', {
+                    path: `/kibana/settings?query=category:(observability)`,
+                  })}
+                >
+                  Click here
+                </EuiLink>
```

Comment:
Can you change it to:

> The full list of APM options can be found in [Kibana advanced settings]

Reply:
<img width="793" alt="Screen Shot 2022-10-11 at 9 18 07 AM" src="https://user-images.githubusercontent.com/55978943/195101586-22e7940f-b6e7-45a5-b160-724ac523c8b6.png">